lib/resources/ssl.rb

```diff
@@ -44,6 +46,11 @@ class SSL < Inspec.resource(1)
   def initialize(opts = {})
     @host = opts[:host] ||
             inspec.backend.instance_variable_get(:@hostname)
+    # FIXME: This can be removed when/if @hostname is available as a property for 'Train::Transports::WinRM::Connection'
+    # Train enhancement request for this here: https://github.com/chef/train/issues/128
+    if @host.nil? && inspec.backend.class.to_s == 'Train::Transports::WinRM::Connection'
```

Comment:
Please get this merged first. We can remove it in the second cycle and improve train. I don't think we should block this change because of a minor adjustment.